bin/extract-sourcemap

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+// Extract a sourcemap as JSON from a given source file
+
+let { readFile } = require("fs");
+let regex = /sourceMappingURL=data:application\/json;base64,(\w+)/;
+
+let args = process.argv.slice(2);
+if(args.length !== 1) {
+	console.error("Please provide exactly one filename");
+	process.exit(1);
+}
+
+readFile(args[0], (err, content) => {
+	if(err) {
+		throw err;
+	}
+	let sourceMapAsBase64 = regex.exec(content)[1];
```

Comment:
Should we handle errors (like missing/unrecognised sourceMappingURL line) here as well?